src/rules/rule-no-shorthand-property-overrides/README.md

```diff
@@ -1,5 +1,7 @@
 # rule-no-shorthand-property-overrides
 
+**Deprecated: use the [`declaration-block-no-shorthand-property-overrides`](../declaration-block-no-shorthand-property-overrides/README.md) rule instead.**
+
```

Comment:
Just double-checking ... did you remember to put this note at the top of the other readmes?